resources/qml/SearchBar.qml

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
```

Comment:
This looks like a re-usable component. Shouldn't we also make sure that this component is used in other places where we have a search bar (settings, digital factory)

Reply:
Yes that was the intention. See the changes to '26 plugins/DigitalLibrary/resources/qml/SelectProjectPage.qml' in the 1st commit. But I'll try to add it to the settings tomorrow as well :-)

Reply:
Done in https://github.com/Ultimaker/Cura/pull/10812/commits/87f94e680bf0da3f96d03e2a72c6848cf7451c3c